src/ucp/core/ucp_ep.c

```diff
@@ -1033,6 +1033,16 @@ static void ucp_ep_config_set_am_rndv_thresh(ucp_worker_h worker, uct_iface_attr
               config->tag.rndv.am_thresh, config->tag.rndv_send_nbr.am_thresh);
 }
 
+static void ucp_ep_config_adjust_max_short(ssize_t *max_short,
```

Comment:
instead of this function
`max_short = ucs_min(max_short, (ssize_t)rndv_thresh - 1)`

Reply:
@yosefe it wouldn't work in case of `rndv_thresh > SSIZE_MAX`

Reply:
@yosefe do you agree?

Reply:
how about `max_short = ucs_min(max_short + 1, rndv_thresh) -1`

Reply:
imo, previous construction was easier to read